tvx/main.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"sort"
+	"strings"
+
+	"github.com/filecoin-project/lotus/api"
+	"github.com/filecoin-project/lotus/api/client"
+	"github.com/multiformats/go-multiaddr"
+	manet "github.com/multiformats/go-multiaddr-net"
+	"github.com/urfave/cli/v2"
+)
+
```

Comment:
define flags common to multiple sub-commands (like fileFlag) here rather than in one of the multiple commands using them?

Reply:
Common flags like API are fine to define here (I had them in `flags.go`), but flags like `--file` should not have a common definition because we won't be able to contextualise their usage strings, e.g. 

* `--file` in `exec-lotus` is the input vector.
* `--file` in `extract-msg` is where the vector will be output.

I've localized such flag definitions because they're actually not supposed to be shared.